public/assets/images/x-mark.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
you should add the public folder `/public` to your `.gitignore` so that Git doesn't track all these changes to your public assets. It's usually a best practice and just makes the process of reviewing PRs easier.

Reply:
We don't want her `index.html` to be ignored though, so it should be `public/assets` that's ignored.

Reply:
ahh yes, you're right!